docker-compose/monitor/jaeger-v2-config.yml

```diff
@@ -0,0 +1,48 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter, spanmetrics]
+    metrics/spanmetrics:
+      receivers: [spanmetrics]
+      exporters: [prometheus]
+  telemetry:
+    logs:
+      level: DEBUG
+
+extensions:
+  jaeger_query:
+    trace_storage: some_storage
+    metric_storage: some_metrics_storage
+  jaeger_storage:
+    backends:
+      some_storage:
+        memory:
+          max_traces: 100000
+    metric_backends:
+      some_metrics_storage:
+        prometheus:
+          endpoint: http://prometheus:9090
+          normalize_calls: true
+          normalize_duration: true
```

Comment:
I am not sure if these boolean flags are necessary, we should verify if it works without them. Perhaps we don't need them as options are all.

@albertteoh I see that connector config allows "namespace", so makes sense that we have it too. But where do these normalize functions come from? E.g. the calls metric becomes `calls_total` - is it _ever_ different than that?